notes/v1.3.0-RC1.md

```diff
@@ -1,3 +1,384 @@
-# bloop `v1.3.0-RC1`
+# bloop `v1.3.0`
 
-To be done.
\ No newline at end of file
+Bloop is a [build server and CLI tool for the Scala programming language]() developed by the Scala Center.
+
+Bloop `v1.3.0` is the biggest release of this project to this date.
+
+With over 520 commits since the `v1.2.5` release, ~26000 additions and ~9500
+deletions, it aggregates together fundamental changes in the compilation
+pipeline, novel build server features, user-facing improvements and bug fixes
+together.
+
+## Upgrade guide :inbox_tray:
+
+These are **only** the upgrade steps. If you don't have bloop installed, you must read
+the [installation instructions][] instead.
+
+If you're on macOS, **upgrade** to the latest version with:
+
+```sh
+$ brew upgrade scalacenter/bloop/bloop
+$ brew services restart bloop
+```
+
+If you're on Windows using `scoop`, **upgrade** to the latest version with:
+
+```sh
+$ scoop upgrade bloop
+$ bloop ng-stop
+$ // Start the server as you usually do (via systemd, manually, desktop services)
+```
+
+:new: If you're on Arch Linux, **upgrade** to the latest version with:
+
+```sh
+$ rm -rf ~/.bloop
+$ yaourt -S bloop
+$ systemctl --user start bloop
+```
+
+Otherwise, use the generic installation method:
+
+```
+$ curl -L https://github.com/scalacenter/bloop/releases/download/v1.2.5/install.py | python
+$ bloop ng-stop
+$ // Start the server as you usually do (via systemd, manually, desktop services)
```

Comment:
maybe write the alternatives here?

Reply:
I've finally used systemctl as the default, thanks for the suggestion :+1: